### PR TITLE
Update benchmark.py

### PR DESCRIPTION
**Pull request by DevyRuxpin**
suggested improvements. I'm early in my development career....

### CODE REVIEW

File: benchmark.py [hunk 1]
```diff
@@ -91,43 +99,31 @@ def f1_score(prediction, ground_truth):
...
     )
     return res
 
-def run_agent(level = 'easy'):
-    os.environ["AZURE_BASE_URL"] = ""
-    os.environ["AZURE_DEPLOYMENT_NAME"] = ""
-    os.environ["AZURE_MODEL_NAME"]="gpt4"
-    os.environ["AZURE_OPENAI_API_VERSION"]=""
-    os.environ["AZURE_OPENAI_API_KEY"]=  ""
+def run_agent(level='easy'):
```
Comment (tarun-aiplanet) [inline benchmark.py:121]: revert the changes for this function

Comment (DevyRuxpin) [reply benchmark.py]: I have reverted. Hopefully....Any other problems let me know. Thanks for taking the time with me on this.
